shell/platform/android/io/flutter/embedding/engine/systemchannels/SplitAotChannel.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.systemchannels;
+
+import android.os.Build;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import io.flutter.FlutterInjector;
+import io.flutter.Log;
+import io.flutter.embedding.engine.dart.DartExecutor;
+import io.flutter.embedding.engine.dynamicfeatures.DynamicFeatureManager;
+import io.flutter.plugin.common.JSONMethodCodec;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class SplitAotChannel {
```

Comment:
Just coming in early to comment on the name. Can we s/SplitAot/DeferredFeature or DeferredModule here and elsewhere in docs and instructions etc? Since "split AOT" is a strict subset of what we're really shipping.